services/uploads/src/antivirus.js

```diff
@@ -17,6 +18,11 @@ const constants = require('./constants');
  * @return {int} Length of S3 object in bytes.
  */
 async function sizeOf(key, bucket) {
+    console.log('key: ' + key);
+    console.log('bucket: ' + bucket);
+    var sts = new AWS.STS();
```

Comment:
are either of these two varaibles used?

Reply:
They don't seem to be -- I just copied the files over, but yeah I think we can delete these.